### PR TITLE
[UXE-2730] fix: right menu does not close when scrolling

### PR DESCRIPTION
**Pull request by gablisb0a**
### What is the new behavior introduced by this PR?
The only way I found to inject CSS was by using inline styling, via the prop itself.

Why did I use inline styling? By default, this component uses "position: absolute". If I globally changed it to "position: fixed", it would impact other use cases of the button, so I dismissed that option. Using "position: absolute" through inline styling gains more importance, even if I have to use "!important" to ensure it takes precedence. Therefore, in this instance of the component, the only option left to me was inline styling, which will have higher priority and allows the use of "position: fixed" at this level. This resolves the issue.

### Does this PR introduce breaking changes?
- [X] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
https://github.com/aziontech/azion-console-kit/assets/11697561/5b54800b-fa02-4a91-85fb-3e746d9c4cdb

### Does it have a link on Figma?
No, no Figma

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [X] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [X] Commits are tagged with the right word (fix, feat, test, etc)
- [ ] User inputs are sanitized/protected against malicious attacks
- [X] Tags are added to the PR

#### These changes were tested on the following browsers:
- [X] Chrome
- [X] Edge
- [X] Firefox
- [X] Safari

### CODE REVIEW

**Comment by peterpaulo-azion** [comment]
Ajustar o titulo do PR: [ISSUE_CODE] TYPE: TITLE
image

File: src/templates/profile-block/index.vue [hunk 1]
```diff
@@ -108,6 +109,7 @@
   </Sidebar>
 
   <PrimeMenu
+    style="position: fixed"
```
Comment (peterpaulo-azion) [inline src/templates/profile-block/index.vue:112]: tentou usar?

```suggestion
   class="!fixed"
```

**Review by william-tome — APPROVED** [review]

File: src/templates/profile-block/index.vue (hunk 1 above)
Comment (william-tome) [reply src/templates/profile-block/index.vue:112]: Me intrometendo no assunto. Essa sugestão não funcionaria, porque como o Gab explicou, o componente da prime já usa o position como inline style por padrão.

Comment (gablisb0a) [reply src/templates/profile-block/index.vue:112]: @william-tome / @peterpaulo-azion Exato, quando eu faço isso o componente quebra, pois fica como se tivessemos passado 2 positions. O o unico metodo que funciona é o inline mesmo.

**Comment by gablisb0a** [comment]
@peterpaulo-azion / @william-tome Obrigado pelo Review, assim que liberarmos o deploy, farei o merge.

**Pull request closed without merging**